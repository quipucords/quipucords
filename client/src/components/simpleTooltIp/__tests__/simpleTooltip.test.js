import React from 'react';
import { mount } from 'enzyme';
import SimpleTooltip from '../simpleTooltip';

describe('SimpleTooltip Component', function() {
  it('should render', () => {
    const props = {
      id: 'test'
    };

    const component = mount(<SimpleTooltip {...props}>Test tooltip</SimpleTooltip>);

    expect(component.render()).toMatchSnapshot();
  });
});
