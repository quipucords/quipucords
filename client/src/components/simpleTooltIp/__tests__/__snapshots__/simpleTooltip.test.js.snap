// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimpleTooltip Component should render 1`] = `
<span>
  Test tooltip
</span>
`;
